.env

```diff
@@ -2,15 +2,7 @@ FLASK_APP=run.py
 SQLALCHEMY_DATABASE_URI=postgresql://user_name:change_password@resources-psql:5432/resources-psql
 POSTGRES_USER=user_name
 POSTGRES_PASSWORD=change_password
-POSTGRES_DB=postgres
-DB_HOST=postgresql
-DB_PORT=5432
-DB_DATABASE=resources-psql
-DB_USERNAME=user_name
-DB_CONNECTION=pgsql
-USER=user_name
-PASSWORD=change_password
-DB=resources-psql
+POSTGRES_DB=resources-psql
 FLASK_SKIP_DOTENV=1
```

Comment:
Can FLASK_SKIP_DOTENV be set from a config instead of environment variable?

Reply:
http://flask.pocoo.org/docs/1.0/cli/#disable-dotenv

Reply:
So no it can't, and no we can't get this to work by adjusting the dockerfile to remove the `.env` and by adding the files other than `.env`

Attempting to set these application level environments in the dockerfile by using the `ENV` command.

Reply:
Set it in the Dockerfile